pkg/base1/cockpit.js

```diff
@@ -22,7 +22,7 @@
  * Newer code should not include cockpit.css, but our examples still do
  */
 import "../lib/patternfly/patternfly-cockpit.scss";
-import "@patternfly/patternfly/components/_all.scss";
+import "@patternfly/patternfly/patternfly.scss";
```

Comment:
This is what causes the wrong font paths to creep in. I tried in vain (so far) to replace this with a set of different includes which would cover "all PF components except for the global bit that pull in the fonts", but that doesn't really work: It is node_modules/@patternfly/patternfly/components/_all.scss itself which requires `$pf-global--breakpoint--sm` and that is only available in `_global-variables.scss`. (In other words, `_all.scss` does not properly import its own dependencies). We have no control over that file. sass-resources-loader hacks around this by including this and other files absolutely everywhere, but that is really just a hack. Also, I suppose files that start with an underscore are not meant to be API?

So IMHO the cleanest way would be to use patternfly.scss as in this patch, and then apply our font cleaning somehow.